src/main/java/com/vladyslavpalamarchuk/trackmycoin/adaptors/telegram/TelegramBotConsumer.java

```diff
@@ -12,9 +14,13 @@
 @RequiredArgsConstructor
 public class TelegramBotConsumer implements LongPollingSingleThreadUpdateConsumer {
 
+  private final CommandProcessorRegistry commandProcessorRegistry;
+
   @Override
   public void consume(Update update) {
     if (update.hasMessage() && update.getMessage().hasText()) {
+      CommandProcessor processMessage = commandProcessorRegistry.get(update.getMessage().getText());
```

Comment:
Please review name of the field, looks like incorrect and should be commandProcessor